config:
  frame:
    parent: world
    child: magnet
  motion:
    update_rate_hz: 100.0
    magnetic_strength: 5.0
  tf:
    enable: true
    static: false
  topics:
    pose: /magnetic/pose_true
  trajectory:
    type: circular
    center: [0.0, 0.0, 0.05]
    radius: 0.1
    angular_speed: 0.5
    phase: 0.0
    orientation_rpy: [0.0, 0.0, 0.0]
